modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivitySendEmailWorker.php

```diff
@@ -170,26 +170,31 @@ public function processItem($data) {
         $recipients = $data['recipients'];
       }
 
-      // Load the user accounts.
-      $target_accounts = $user_storage->loadMultiple($recipients);
+      /** @var \Drupal\user\UserInterface[] $target_accounts */
+      $target_accounts = User::loadMultiple($recipients);
```

Comment:
Hi @ribel 

I was thinking, wouldn't it be better to use a load by properties here?

Imagine we are loading 50 users here and 40 of them have not logged in, that wouldn't make sense to itterate over them if we can already rule them out beforehand.

Similarly what we will do in the near future is to load by properties to onl load users who have the frequency set to "immediatly"

Reply:
@ronaldtebrake Could you elaborate on that?
I am not sure if we can pass an array with $recipients user ids to loadByProperties() method.